paasta_tools/metrics/metastatus_lib.py

```diff
@@ -736,6 +735,12 @@ def format_table_column_for_healthcheck_resource_utilization_pair(healthcheck_ut
             naturalsize(healthcheck_utilization_pair[1].total * 1024 * 1024, gnu=True),
             utilization_perc,
         ))
+    elif humanize and healthcheck_utilization_pair[1].metric in ['cpus', 'gpus']:
```

Comment:
Do we ever not humanize?

Let's just humanize all the time and remove the complexity?